package.json

```diff
@@ -2,36 +2,26 @@
   "name": "range-specifier-parser",
   "version": "1.0.2",
   "description": "Range Specifier Parser",
-  "homepage": "https://github.com/seegno/range-specifier-parser",
+  "homepage": "https://github.com/uphold/range-specifier-parser",
+  "repository": "git@github.com:uphold/range-specifier-parser",
   "license": "MIT",
-  "author": "Seegno",
-  "main": "dist/src",
-  "repository": "git@github.com:seegno/range-specifier-parser",
+  "author": "Uphold",
+  "main": "src/index.js",
```

Comment:
This change seems to belong to the separate PR of removing babel.

Reply:
This change has been moved to #11.